### ISSUE TITLE
FreeBSD tests failing in CI

### ISSUE THREAD

**Issue by emaste** [lsof-org/lsof#259]
**Describe the bug**
case-13-classic and case-14-classic-opt are failing in Cirrus-CI on FreeBSD 14 snapshots [Tests are also failing on FreeBSD 12 and 13, but let's start with the most up-to-date]

**To Reproduce**
Run tests via Cirrus-CI

**Expected behavior**
Tests pass

**Program output**
```
STARTING TEST ( dialect neutral )
=============================================================================
case-00-hello ... ok
case-01-version ... ok
case-13-classic ... failed
case-14-classic-opt ... failed
case-20-exit-status ... ok
case-20-fd-only-inclusion ... ok
case-20-handle-missing-files ... ok
case-20-offset-field ... ok
case-20-repeat-count ... ok
case-21-exit-Q-status ... ok
```

**Environment (please complete the following information):**

 - Kernel: FreeBSD 14 snapshot
 - OS: FreeBSD
 - lsof Version: git ce84a35f82e8c7af4f1d370bc983001090a78a2d
 - Origin: GCP image
 - 
**Additional context**

Link to test run: https://cirrus-ci.com/task/6574869868445696

**Comment by emaste** [comment]
`ERROR!!!  open lsof /dev/kmem usage wasn't found.` - /dev/kmem is not used any longer on FreeBSD main, so this is unsurprising. This part of the test no longer makes sense after d5555d1006e930a8ebe37551b1c22257dda95ad2

**Comment by jiegec** [comment]
Fixed in pr #271
